src/userportal.config

```diff
@@ -1,5 +1,5 @@
 {
   "debug": true,
-  "applicationContext": "/ovirt-engine",
-  "applicationURL": "/ovirt-engine/web-ui"
+  "applicationContext": "",
```

Comment:
Ah you are right. For some reason I thought that there are 2 separate userportal.config files used for dev and production but now I notice that this will indeed end up being copied in the resulting war. 

I'm thinking that to differentiate between dev and prod we can use the presence of this config file, e.g. if we fail to load the config -> the default (development) will be used..